locales/en/npc.json

```diff
@@ -37,8 +37,8 @@
     "expPointsText": "The yellow bar tracks your avatar's experience points. Whenever you succeed in achieving a task, you gain both gold and experience. When your experience bar maxes out, you gain a level. Gaining levels is how you unlock new and exciting features on HabitRPG.",
     "typeGoals": "Types of Tasks",
     "typeGoalsText": "HabitRPG allows you to track your tasks in three different ways. These tasks are categorized in columns as Habits, Dailies, or To-Dos.",
-    "tourHabits": "Habits are tasks that you constantly track. They can be given plus or minus values, allowing you to gain experience and gold for good habits or lose health for bad ones.",
-    "tourDailies": "Dailies are tasks that you want to complete once a day. Checking off a Daily reaps experience and gold. Failing to check off your daily before the day resets results in a loss of health. You can change your day start settings from the Settings menu (click the gear-shaped icon, then click \"Site\").",
+    "tourHabits": "Habits are tasks that you constantly track. They can be given plus or minus values, allowing you to gain experience and gold for good Habits or lose health for bad ones.",
```

Comment:
I went through the tour again with the updated changes and this line stood out: "good Habits" gives gold and exp. It seems like this refers to real life habits, so should it be changed to "habits"?